pykwasm/src/pykwasm/kdist/wasm-semantics/ulm-wasm.md

```diff
@@ -194,11 +198,16 @@ In the remote semantics, the Wasm VM has a fixed entrypoint.
 Passing Control
 ---------------
 
-The embedder loads the module to be executed and then resolves the entrypoint function.
+The embedder loads the module to be executed and then calls the entrypoint function.
 
 ```k
-    rule <k> PGM:PgmEncoding => #resolveCurModuleFuncExport(#getEntryPoint()) </k>
-         <instrs> .K => decodePgm(PGM) </instrs>
+    rule
+        <k> PGM:PgmEncoding
+            => setContractModIdx
+                ~> #resolveCurModuleFuncExport(#getEntryPoint())
+                ~> setSuccessStatus
```

Comment:
Hmm... I haven't thought about this to the level of detail that you have! Since this is just a style question anyway, let's keep this as is and we can discuss later the possibility of a K coding style.